rules/falco_rules.yaml

```diff
@@ -2414,7 +2421,9 @@
     (container.image.repository in (gcr.io/google_containers/hyperkube-amd64,
      gcr.io/google_containers/kube2sky, docker.io/sysdig/falco,
      docker.io/sysdig/sysdig, docker.io/falcosecurity/falco,
-     sysdig/falco, sysdig/sysdig, falcosecurity/falco) or (k8s.ns.name = "kube-system"))
+     sysdig/falco, sysdig/sysdig, falcosecurity/falco)
+     or (container.image.repository in (ibm_cloud_containers))
```

Comment:
a list can be embed into another list, so can you change it to `falcosecurity/falco, ibm_cloud_containers`